### ISSUE TITLE
Number Format is not validated

### ISSUE THREAD

**Issue by fescobar** [raml-org/raml-js-parser-2#418]
When format value is invalid, it doesn't appear an error message.
The value MUST be one of the following: int32, int64, int, long, float, double, int16, int8

```
#%RAML 1.0
title: API Animals & People
version: 1.7
baseUri: http://www.sa.uk
mediaType: [ application/json, application/xml ]

types:
  Email:
    type: object
    properties:
      size:
        type: number
        minimum: -3
        maximum: 5
        format: anyType

  Emails:
    type: Email[]
    minItems: 0
    maxItems: 10
    uniqueItems: true
    example: # example that contains array
      - # start item 1
        size: -1
      - # start item 2
        size: -2
```

![numberformat is not validated](https://cloud.githubusercontent.com/assets/1236418/17557375/4f274f24-5eed-11e6-9e77-e70ca3e46d92.png)

**Comment by fescobar** [comment]
Neither the value is validated.

![numberformat is not validated2](https://cloud.githubusercontent.com/assets/1236418/17558223/42577e1a-5ef0-11e6-89fd-505b1ce386bc.png)

**Comment by ddenisenko** [comment]
Fixed in hotfix-for-0.2.31 branch of typesystem.
